tests/tests_integration/test_api/test_data_modeling/test_views.py

```diff
@@ -79,6 +79,17 @@ def test_delete_non_existent(self, cognite_client: CogniteClient, integration_te
             == []
         )
 
+    def test_retrieve_without_inherited_properties(self, cognite_client: CogniteClient, cdf_views: ViewList) -> None:
+        assert len(cdf_views) >= 1, "Please add at least one view to the test environment"
+        # Arrange
+        view = cdf_views[0]
+
+        # Act
+        retrieved = cognite_client.data_modeling.views.retrieve(view.as_id(), include_inherited_properties=False)[0]
+
+        # Assert
+        assert retrieved is not None
```

Comment:
Above would raise KeyError if missing, so this won't be reached.
But let's actually assert that at least one of the properties are inherited (there is a flag on the property def which indicates if it's inherited or not) and vice versa when include_inherited_properties is False.

Reply:
I cannot see that flag in the spec. Which flag are you referring to?

Reply:
Hmm. Doesn't look like that's actually exposed through DMS, only in the internal API. Then this is fine for now.